lib/hyrax/transactions/steps/set_embargo.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+require 'dry/monads'
+
+module Hyrax
+  module Transactions
+    module Steps
+      ##
+      # Adds and removes work members
+      class SetEmbargo
+        include Dry::Monads[:result]
+
+        ##
+        # @param [Hyrax::Work] obj
```

Comment:
the docs look out of date. i prefer `obj` as the first argument on transaction steps, in general. but i guess i don't object to `work_resource` as long as the docs are updated to match